### ISSUE TITLE
Changing skyType breaking in aframe 0.8.2

### ISSUE THREAD

**Issue by aj019** [supermedium/aframe-environment-component#54]
I tried changing the environment's skyType from default atmosphere to color in two versions of aframe.

In Aframe 0.6.1 
- The skyType changes to color and the result is seen in the scene as well.

In Aframe 0.8.2
- The sky becomes black and stops responding to any change in skyColor or horizonColor

**Comment by feiss** [comment]
I've seen unexpected behaviour in this sense using the inspector, but not on the actual experience. Do you have this problem while using inspector?

**Comment by aj019** [comment]
Yes i am experiencing this behaviour while i am changing the skyType using the inspector.
